<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreciseOutputWindow</class>
 <widget class="QDialog" name="PreciseOutputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>High precision output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="suffix">
        <string> digits</string>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progressCircleLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="evaluateButton">
       <property name="text">
        <string>Evaluate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
